.github/workflows/build_wheels_linux.yml

```diff
@@ -144,8 +147,17 @@ jobs:
           fi
           echo "::endgroup::"
 
-      - uses: atalman/checkout-action@main
+      - name: install nodejs20glibc2.17
+        if:  ${{ inputs.architecture == 'x86_64' }}
+        run: |
+          yum install curl -yyq
+          curl -LO https://unofficial-builds.nodejs.org/download/release/v20.9.0/node-v20.9.0-linux-x64-glibc-217.tar.xz
```

Comment:
If possible, let's host this on our S3 bucket as I'm not sure how reliably this host is, we might end up ddos it

Reply:
I think https://github.com/actions/runner/issues/2906#issuecomment-2109514798 says that rehosting the file is not what nodejs allows people to do, so it's better to just stick with this link then